### PR TITLE
doc: update examples to explain ghc-component and aliases

### PR DESCRIPTION
**Pull request by frasertweedale**
---

## Advisory

- [ ] It's not duplicated
- [ ] All fields are filled
- [ ] It is validated by `hsec-tools`

## hsec-tools

- [ ] Previous advisories are still valid

### CODE REVIEW

**Review by blackheaven — APPROVED** [review]

**Review by mihaimaruseac — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -73,6 +78,9 @@ url = "https://github.com/username/package/pull/139"
 
 # Mandatory: name of the affected package on Hackage
 package = "acme-broken"
+# OR: name of the affected GHC component; one of:
+# ghc | ghci | rts | ghc-pkg | runghc | ghc-iserv | hp2ps | hpc | hsc2hs | haddock
+ghc-component = "ghci"
```
Comment (TristanCacqueray) [inline README.md:83]: The README is also validated by the CI, so this key should be commented.

Comment (frasertweedale) [reply README.md:83]: Thank you, I'll fix it now.

**Comment by frasertweedale** [comment]
@blackheaven CI nix job is failing due to out of disk error:
```
write /nix/store/q3y9jmlsfkpk0kgpc6spvkgk3hgsxphh-ghc-9.2.8/lib/ghc-9.2.8/haskeline-0.8.2/libHShaskeline-0.8.2-ghc9.2.8.so: no space left on device
```

Do you have time to look into it?

**Comment by blackheaven** [comment]
I'll have a look

**Comment by mihaimaruseac** [comment]
The Nix job passed now, merging this one

**Pull request merged**